config/armsve/bli_cntx_init_armsve.c

```diff
@@ -65,20 +65,22 @@ void bli_cntx_init_armsve( cntx_t* cntx )
 	);
 
 	// Set VL-specific packing routines if applicable.
+	// NOTE: SVE-Intrinsic kernels are used without checking __has_include(<arm_sve.h>).
+	//  Such is ensured at configuration stage for config: armsve.
 	if (m_r_d==16)
 	  bli_cntx_set_packm_kers
 	  (
 		3,
 		BLIS_PACKM_10XK_KER, BLIS_DOUBLE, bli_dpackm_armsve512_asm_10xk,
-		BLIS_PACKM_12XK_KER, BLIS_DOUBLE, bli_dpackm_armsve512_asm_12xk,
+		BLIS_PACKM_12XK_KER, BLIS_DOUBLE, bli_dpackm_armsve512_int_12xk,
```

Comment:
Is the 12xk kernel actually used in this config? If not it can be moved to the "old" directory.

Reply:
Unused in fact.